src/Microsoft.AspNet.Server.Kestrel/Http/Listener.cs

```diff
@@ -53,7 +53,7 @@ public Listener(IMemoryPool memory)
                     ListenSocket = new UvTcpHandle();
                     ListenSocket.Init(Thread.Loop, Thread.QueueCloseHandle);
                     ListenSocket.Bind(new IPEndPoint(IPAddress.Any, port));
-                    ListenSocket.Listen(10, _connectionCallback, this);
+                    ListenSocket.Listen(128, _connectionCallback, this);
```

Comment:
The number of listeners is based on the processor count. Is there anything better than a magic number that's also repeated a couple of times later?